Userland/Applications/PixelPaint/Tools/MoveTool.h

```diff
@@ -27,17 +33,16 @@ class MoveTool final : public Tool {
 
 private:
     virtual StringView tool_name() const override { return "Move Tool"sv; }
-    ErrorOr<void> update_cached_preview_bitmap(Layer const* layer);
+    ErrorOr<void> update_cached_preview_bitmap(Layer const*);
+    Optional<Corner const> mouse_in_resize_corner(Layer const*, MouseEvent&);
```

Comment:
Same, I would try to find a better name for this function. When I read it I think it returns a boolean

Reply:
Changed to `resize_anchor_location_from_cursor_position`. I don't love it, as its a bit verbose, but at least its descriptive.